tsdb/head_wal.go

```diff
@@ -460,6 +475,292 @@ func (wp *walSubsetProcessor) processWALSamples(h *Head, mmappedChunks map[chunk
 	return unknownRefs, mmapOverlappingChunks
 }
 
+func (h *Head) loadWbl(r *wal.Reader, multiRef map[chunks.HeadSeriesRef]chunks.HeadSeriesRef, lastMmapRef chunks.ChunkDiskMapperRef) (err error) {
```

Comment:
```suggestion
func (h *Head) loadWBL(r *wal.Reader, multiRef map[chunks.HeadSeriesRef]chunks.HeadSeriesRef, lastMmapRef chunks.ChunkDiskMapperRef) (err error) {
```

Reply:
This function needs some readability love too

Reply:
TODO for me, review this later. My energy had depleted ;p

Reply:
True XD however it is much simpler than loadWAL